examples/powertools-examples-core/serverless/serverless.yml

```diff
@@ -0,0 +1,41 @@
+service: hello
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: java11
+
+# you can overwrite defaults here
+#  stage: dev
+#  region: us-east-1
+
+# you can define service wide environment variables here
+  environment:
+    POWERTOOLS_LOG_LEVEL: INFO
+    POWERTOOLS_LOGGER_SAMPLE_RATE: 0.1
+    POWERTOOLS_LOGGER_LOG_EVENT: true
```

Comment:
we don't support this variable, I know we have it everywhere but it's not handled in the code, can you remove (we'll need to remove it from everywhere in v1.x)

Reply:
Sure, I removed it and, and will create an issue to remove it everywhere else as well